src/helpers/shortcodeLayoutParser.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const AMP_LAYOUTS = new Set([
+  'nodisplay',
+  'fixed',
+  'responsive',
+  'fixed-height',
+  'fill',
+  'container',
+  'flex-item',
+  'fluid',
+  'intrinsic',
+]);
```

Comment:
there is no place that this list can be imported from (can the validator expose something?) Would be great to avoid an orphaned list in case the available layouts changes in the future and we forget to update it

Reply:
Good point. They could be imported be inferred via the validator rules, but this feels a bit overkill (once has to walk over all tags and check the supported layouts). Layouts change very seldom so I think it's OK to hard-code.